Cargo.toml

```diff
@@ -26,7 +26,7 @@ parking_lot = "0.12"
 bytemuck = { version = "1.12.1", features = ["derive"] }
 smallvec = "1.8.0"
 moscato = { git = "https://github.com/dfrg/pinot" }
-peniko = { git = "https://github.com/linebender/peniko" }
+peniko = { git = "https://github.com/linebender/peniko", rev = "ba96b90" }
```

Comment:
Do we want to pin the one above as well?

Reply:
We can but I don't plan any further updates to moscato and it will go away as soon as googlefonts/fontations#191 lands.

Reply:
Added a pin just for consistency. Doesn't hurt!